packages/components/note/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "@tradeshift/elements.note",
+  "version": "0.3.3",
+  "src": "src/note.js",
+  "main": "lib/note.umd.js",
+  "browser": "lib/note.umd.js",
+  "files": [
+    "lib/*"
+  ],
+  "dependencies": {
+    "@tradeshift/elements": "^0.3.3",
+    "@tradeshift/elements.icon": "file:../icon"
```

Comment:
I am not sure, do you need npm version or file. 
What about button dependency

Reply:
I forgot to add the button, good catch 👍 
I am still confused about this one! In the development process it's easier to just use the file but it's a bad practice I think!
We need to get into automating the publish process, and with that the publish process is headache free so we can actually fix the required feature or fix in different PR! In that situation having the npm version works just fine!